src/osd/PrimaryLogPG.h

```diff
@@ -840,6 +840,10 @@ class PrimaryLogPG : public PG, public PGBackend::Listener {
       // requeue at front of scrub blocking queue if we are blocked by scrub
       for (auto &&p: to_req) {
 	if (scrubber.write_blocked_by_scrub(p.first.get_head())) {
+          for (auto op : p.second) {
```

Comment:
might want to use `auto&` to avoid the `intrusive_ptr_add_ref` and `intrusive_ptr_release` calls.

Reply:
ok，that's right